app/views/hyrax/dashboard/collections/_form_share.html.erb

```diff
@@ -9,54 +9,61 @@
         <% access_options = options_for_select([['Manager', 'manage'], ['Depositor', 'deposit'], ['Viewer', 'view']]) %>
 
         <%= simple_form_for @form.permission_template,
-          url: [hyrax, :dashboard, @form, :permission_template],
-          html: { id: 'group-participants-form' } do |f| %>
+                            url: [hyrax, :dashboard, @form, :permission_template],
+                            html: { id: 'group-participants-form' } do |f| %>
+          <div class="clearfix spacer">
+            <%= f.fields_for 'access_grants_attributes',
+                             f.object.access_grants.build(agent_type: 'group'),
+                             index: 0 do |builder| %>
 
-          <%= f.fields_for 'access_grants_attributes',
-            f.object.access_grants.build(agent_type: 'group'),
-            index: 0 do |builder| %>
+              <div class="form-inline add-sharing-form">
+                <label class="col-md-2 col-xs-4 control-label"><%= t('.add_group') %>:</label>
+                <div class="col-md-10 col-xs-8 form-group">
+                  <%= builder.hidden_field :agent_type %>
+                  <%= builder.text_field :agent_id,
+                                         placeholder: "Search for a group...",
+                                         class: 'form-control search-input' %>
+                  as
+                  <%= builder.select :access,
+                                     access_options,
+                                     { prompt: "Select a role..." },
+                                     class: 'form-control' %>
 
-            <div class="form-inline add-sharing-form">
-              <div class="form-group">
-                <label><%= t('.add_group') %>:</label>
-                <%= builder.hidden_field :agent_type %>
-                <%= builder.text_field :agent_id, placeholder: "Search for a group...", class: 'form-control search-input' %>
-                as
-                <%= builder.select :access, access_options,
-                { prompt: "Select a role..." },
-                class: 'form-control' %>
-
-                <%= f.submit t('helpers.submit.hyrax_permission_template_access.create'), class: 'btn btn-info' %>
+                  <%= f.submit t('helpers.submit.hyrax_permission_template_access.create'), class: 'btn btn-info' %>
+                </div>
               </div>
-            </div>
-
-          <% end %>
+            <% end %>
+          </div>
         <% end %>
 
         <%= simple_form_for @form.permission_template,
-          url: [hyrax, :dashboard, @form, :permission_template],
-          html: { id: 'user-participants-form' } do |f| %>
+                            url: [hyrax, :dashboard, @form, :permission_template],
+                            html: { id: 'user-participants-form' } do |f| %>
+          <div class="clearfix spacer">
```

Comment:
Are these `<div class="clearfix spacer">`s necessary?

Reply:
Ah, @jgondron just saw your comment about adding some space between rows.  I get it.  If the Add group `<form>` was rendering properly, then could probably target that form class with some CSS to add some bottom margin/padding instead of adding extra markup to get the spacing, but can always revisit that later.